tools/sense_studio/video_recording.py

```diff
@@ -14,6 +15,12 @@
 video_recording_bp = Blueprint('video_recording_bp', __name__)
 
 
+@video_recording_bp.route('/ffmpeg-check', methods=['POST'])
```

Comment:
The methods can be changed to `GET` as you are just requesting data here.

Reply:
> The methods can be changed to `GET` as you are just requesting data here.

Thanks and I update this point.